core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -85,6 +86,41 @@ object Invoker {
       abort()
     }
 
+    val proposedInvokerId: Option[Int] = args.headOption.map(_.toInt)
+    val assignedInvokerId = proposedInvokerId
+      .map { id =>
```

Comment:
is this just a foreach?

Reply:
Nope, get's `getOrElse`d below. Unfortunately, `Option` has no method to just apply a side-effect, passing the original value through (like `andThen` on Futures)